website/docs/guides/markdown-features/markdown-features-links.md

```diff
@@ -0,0 +1,52 @@
+---
+id: links
+description: Links to other pages in Markdown
+slug: /markdown-features/links
+---
+
+# Markdown links
+
+There are two ways of adding a link to another page: through a **URL path** and a **file path**.
+
+```md
+- [URL path to another document](./installation)
+- [file path to another document](./installation.md)
+```
+
+URL paths are unprocessed by Docusaurus, and you can see them as directly rendering to `<a href="./installation">`, i.e. it will be resolved according to the page's URL location, rather than its file-system location.
+
+If you want to reference another Markdown file **included by the same plugin**, you could use the relative path of the document you want to link to. Docusaurus' Markdown loader will convert the file path to the target file's URL path (and hence remove the `.md` extension).
+
+For example, if you are in `docs/folder/doc1.md` and you want to reference `docs/folder/doc2.md`, `docs/folder/subfolder/doc3.md` and `docs/otherFolder/doc4.md`:
+
+```md title="docs/folder/doc1.md"
+I am referencing a [document](doc2.md).
+
+Reference to another [document in a subfolder](subfolder/doc3.md).
+
+[Relative document](../otherFolder/doc4.md) referencing works as well.
+```
+
+Relative file paths are resolved against the current file's directory. Absolute file paths, on the other hand, are resolved relative to the **content root**, usually `docs/`, `blog/`, or [localized ones](../../i18n/i18n-tutorial.md) like `i18n/zh-Hans/plugin-content-docs/current`.
+
+Absolute file paths can also be relative to the site directory. However, beware that links that begin with `docs/` or `blog/` are **not portable** as you would need to manually update them if you create new doc versions or localize them.
```

Comment:
It's kind of vague. There are three kinds of paths: `./docs`/`../docs`, `docs/`, and `/docs/`. (1) is definitely relative (and will be resolved relative to current file), (3) is definitely absolute (and will be resolved relative to content root), but (2) can be either (and I don't have a name for this pattern). Therefore, we actually all ALL resolutions for `docs/` paths: it can be seen as relative OR absolute.

Reply:
ok why not 👍